mne_bids_pipeline/_config.py

```diff
@@ -416,6 +416,14 @@
     ```
 """
 
+read_raw_bids_verbose: Optional[Literal["error"]] = None
```

Comment:
Could you mention this new setting in the changelog, please?

Reply:
Done!